SignalServiceKit/src/Messages/Interactions/TSOutgoingMessage.h

```diff
@@ -75,11 +103,7 @@ typedef NS_ENUM(NSInteger, TSGroupMetaMessage) {
 + (instancetype)outgoingMessageInThread:(nullable TSThread *)thread
                        groupMetaMessage:(TSGroupMetaMessage)groupMetaMessage;
 
-@property (atomic, readonly) TSOutgoingMessageState messageState;
-
-// The message has been sent to the service and received by at least one recipient client.
-// A recipient may have more than one client, and group message may have more than one recipient.
-@property (atomic, readonly) BOOL wasDelivered;
+@property (readonly) TSOutgoingMessageState messageState;
```

Comment:
This is now just a property calculated on the fly.  We consult this property a lot, so I might return to backing with an ivar that serves as a cache.